hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHoodieDeltaStreamer.java

```diff
@@ -81,6 +84,7 @@
  * upserts, inserts. Check counts at the end.
  */
 public class TestHoodieDeltaStreamer extends UtilitiesTestBase {
+
   private static final String PROPS_FILENAME_TEST_SOURCE = "test-source.properties";
   private static final String PROPS_FILENAME_TEST_INVALID = "test-invalid.properties";
   private static volatile Logger log = LogManager.getLogger(TestHoodieDeltaStreamer.class);
```

Comment:
Would be changed to final?

Reply:
@leesf : Thanks for your suggestion. We are not following this convention w.r.t test-classes. If you have references to published conventions that we are missing, happy to hear about them :)